test/helper.js

```diff
@@ -0,0 +1,37 @@
+var helper = require('../lib/helper');
+var expect = require('chai').expect;
+
+describe('helper', function() {
+  describe('parseLoopBackVersion()', function() {
+    it('should return valid loopback version', function() {
+      var version = helper.parseLoopBackVersion('2.1.3');
```

Comment:
While testing a dependency version specified as exactly a single version may be useful, I think it's even more important to test version ranges: `^2.1.3`, `~2.1.3`, `2.x`, `2.1.x`.